### ISSUE TITLE
Blog Post: How to test Ember Data Adapters and Serializers

### ISSUE THREAD

**Issue by Turbo87** [mainmatter/mainmatter.com#406]
The testing blueprints in the `ember-data` project are almost empty and don't give any hints on how to test these components. Using the public `store` service API it is quite straight-forward to test them though. Since it is currently not documented anywhere we could write a blog post on it to give people an idea on how to do it.

**Comment by marcoow** [comment]
Yes, I agree this is a good topic and something many people struggle with 👍
